src/sbin/livemedia-creator

```diff
@@ -690,16 +690,26 @@ def anaconda_cleanup(dirinstall_path):
     Cleanup any leftover mounts from anaconda
 
     :param str dirinstall_path: Path where anaconda mounts things
+    :returns: True if cleanups were successful. False if any of them failed.
 
     If anaconda crashes it may leave things mounted under this path. It will
     typically be set to /mnt/sysimage/
+
+    Attempts to cleanup may also fail. Catch these and continue trying the
+    other mountpoints.
```

Comment:
No, that's telling you what it will do. It used to just crash with a CalledProcessError and exit. Now it will try all the mountpoints, even if some of them fail the cleanup attempt.